llarp/rpc/lokid_rpc_client.cpp

```diff
@@ -0,0 +1,202 @@
+#include <rpc/lokid_rpc_client.hpp>
+
+#include <util/logging/logger.hpp>
+
+#include <router/abstractrouter.hpp>
+
+#include <nlohmann/json.hpp>
+
+#include <util/time.hpp>
+#include <util/thread/logic.hpp>
+
+namespace llarp
+{
+  namespace rpc
+  {
+    static lokimq::LogLevel
+    toLokiMQLogLevel(llarp::LogLevel level)
+    {
+      switch (level)
+      {
+        case eLogError:
+          return lokimq::LogLevel::error;
+        case eLogWarn:
+          return lokimq::LogLevel::warn;
+        case eLogInfo:
+          return lokimq::LogLevel::info;
+        case eLogDebug:
+          return lokimq::LogLevel::debug;
+        case eLogNone:
+        default:
+          return lokimq::LogLevel::trace;
+      }
+    }
+
+    LokidRpcClient::LokidRpcClient(LMQ_ptr lmq, AbstractRouter* r)
+        : m_lokiMQ(std::move(lmq)), m_Router(r)
+    {
+      // m_lokiMQ->log_level(toLokiMQLogLevel(LogLevel::Instance().curLevel));
```

Comment:
I assume this is commented out because the log level gets set elsewhere, but if so the above `toLokiMQLogLevel` is unused and can get removed, too.